appuio/stardog/templates/monitoring/stardog-rules.yaml

```diff
@@ -48,7 +48,7 @@ spec:
     {{- if .Values.ingress.enabled }}
     - alert: HttpCheck
       expr: probe_success{instance="https://{{ .Values.ingress.host }}/admin/healthcheck"} != 1
-      for: 1m
+      for: "{{ "{{ if (((.Values.alerts).httpCheck).for) }}" }}{{ .Values.alerts.httpCheck.for }}{{ "{{ else }}" }}1m{{ "{{ end }}" }}"
```

Comment:
I'm not sure but wouldn't something simpler like this work?

{{ default "1m" .Values.alerts.httpCheck.for | quote }}

Reply:
I did try this at first see bc991eebfa2c321ef4da54511da9298d7422aac1, but the tests kept failing with a parse error for "1m", written with and without quotes as well as with pipe to quotes and without.

Reply:
Hmmm I think you should try the way I've proposed (I've used that for rocketchat-deploy therefore I'm pretty sure it works). I'm not convinced that the way you tried does the same thing.